hmy/api_backend.go

```diff
@@ -530,20 +530,21 @@ func (b *APIBackend) GetDelegationsByValidator(validator common.Address) []*stak
 	return delegations
 }
 
-// GetDelegationsByDelegator returns all delegation information of a delegator
-func (b *APIBackend) GetDelegationsByDelegator(
-	delegator common.Address,
+// GetDelegationsByDelegatorByBlockNumber returns all delegation information of a delegator
+func (b *APIBackend) GetDelegationsByDelegatorByBlockNumber(
```

Comment:
name should be `GetDelegationsByDelegatorByBlock`

Reply:
But the other RPCs end in `ByBlockNumber`

Reply:
the ones that end with `ByNumber` are actually sending the Uint64 number, not block. Right?